finalized-tag-updater/src/main/kotlin/net/consensys/linea/FinalizationUpdatePoller.kt

```diff
@@ -32,8 +33,8 @@ data class FinalizationUpdatePollerConfig(
 }
 
 class FinalizationUpdatePoller(
-  vertx: Vertx,
-  config: FinalizationUpdatePollerConfig,
+  private val vertx: Vertx,
+  private val config: FinalizationUpdatePollerConfig,
   private val lineaRollup: LineaRollupV5,
```

Comment:
@jpnovais is it all good to change the above from `LineaRollupV5` to `Web3JLineaRollupSmartContractClientReadOnly`?